docs/content/Checkbox.mdx

```diff
@@ -85,22 +86,59 @@ An `indeterminate` checkbox state should be used if the input value is neither t
 </>
 ```
 
-## Component props
+## Props
 
-Native `<input>` attributes are forwarded to the underlying React `input` component and are not listed below.
+<PropsTable>
+  <PropsTableRow
+    name="checked"
+    type="boolean"
+    description="Modifies true/false value of the native checkbox"
+  />
+  <PropsTableRow
+    name="defaultChecked"
+    type="boolean"
+    description="Checks the input by default in uncontrolled mode"
+  />
+  <PropsTableRow
+    name="onChange"
+    type="(ChangeEvent) => void"
+    description="A callback function that is triggered when the checked state has been changed"    
+  />
+  <PropsTableRow
+    name="disabled"
+    type="boolean"
```

Comment:
Does this have a default value?

Reply:
It's an optional prop, so no. Or are you setting undefined explicitly?

Reply:
Would it make sense to say this defaults to`false`?

Reply:
I think that's misrepresenting the data type, as false !== undefined. I think literally `undefined` would make sense, or to provide a way to clearly indicate it's optional the same way you do with `required`?